deploy/lib/parca-agent/parca-agent.libsonnet

```diff
@@ -283,7 +193,14 @@ function(params) {
           '--metadata-disable-caching',
         ] else []
       ),
-      securityContext: pa.config.securityContext,
+      // Container level security context.
+      securityContext: {
+        privileged: true,
+        allowPrivilegeEscalation: true,
+        capabilities: {
+          add: ['BPF', 'PERFMON'],  // SYS_ADMIN
```

Comment:
Can we maybe first test this on our demo and prod cluster?

Reply:
👍 I will do it. I have tested locally against minikube it worked.

Reply:
This was a good call. In theory, this should work, but this made e2e tests fail in practice.
So we will stick with `SYS_ADMIN` for now. It requires more investigation, which additional capabilities we need besides `BPF` and `PERFMON`

Reply:
Maybe kernel version issue? I think CAP_BPF is only available after 5.8.